### PR TITLE
109 enable safari for e2e

### LINKED ISSUE #109: 🧹 enable safari for e2e
## Description

Running e2e test on CI fails when using Safari

## Affects

- e2e

Update playwright configuration `web/playwright.config.ts`, append the following to the `projects` array

```
    {
      name: 'webkit',
      use: {
        ...devices['Desktop Safari'],
      },
    },
```

CI should start failing once this configuration is enabled, we may need to install additional libraries to get safari running on ubuntu?

### PR DESCRIPTION
**Pull request by iameddieyayaya**
## Story

Include the ID in the branch name and link to the story here

Fix #109 

## Description

Add webkit to playwright configuration
Rewrite edit-profile spec
Replace TextField w/ InputField
Rewrite login/logout spec

## Solution

add test.slow() signup for webkit test to pass

## Was this feature tested on all browsers?

- [x] Chrome
- [x] Firefox
- [x] Safari

## Definition of Done

- [ ] README updated
- [ ] Tests written
- [ ] Acceptance criteria implement
- [ ] Code reviewed
- [ ] Feature accepted

## PR Comments Emoji Legend

😻 - `:heart_cat_eyes:` Compliment to code/idea/etc
♻️ - `:recycle:` Non-blocking proposed refactor
➕ - `:heavy_plus_sign:` Non-blocking proposed addition
🔥 - `:fire:` Non-blocking proposed removal
❓ - `:question:` Non-blocking question
⚠️ - `:warning:` Blocking comment that must be addressed before PR

## Gifs for life (required)

mandatory

### CODE REVIEW

File: redwood.toml [hunk 1]
```diff
@@ -14,4 +14,4 @@
   port = 8911
   host = "${HOST:localhost}"
 [browser]
-  open = true
+  open = false
```
Comment (archae0pteryx) [inline redwood.toml:17]: finally! @craineum this ok? Please say it is... =)

**Review by archae0pteryx — APPROVED** [review]
noice

**Comment by archae0pteryx** [comment]
We should totally update our other tests with the improved code now that we know how to write these things effectively... I bet there's a ton of redundancy in the others like there was with this one... a separate ticket perhaps?

File: redwood.toml (hunk 1 above)
Comment (craineum) [reply redwood.toml:17]: This is what was stopping Safari from working? That it opens a browser window at start?

**Review by craineum — APPROVED** [review]
LGTM!

File: redwood.toml (hunk 1 above)
Comment (craineum) [reply redwood.toml:17]: It would be nice to know for sure, and then share that knowledge. I am sure someone else has hit this issue as well.

**Review by craineum — APPROVED** [review]
Woohoo! Nice work! 🚀

**Pull request merged**